verify/verify.go

```diff
@@ -47,7 +48,9 @@ const (
 var (
 	// ErrMissingVlek is returned when attempting to verify a VLEK-signed report that doesn't also
 	// have its VLEK certificate attached.
-	ErrMissingVlek = errors.New("report signed with VLEK, but VLEK certificate is missing")
+	ErrMissingVlek     = errors.New("report signed with VLEK, but VLEK certificate is missing")
+	workaroundStepping = flag.Bool("workaround_kds_productname", false, "If true, don't compare "+
```

Comment:
If this is default to false, users will still run into issues unless they explicitly set this flag. Shall this be default to true until the issue is addressed?